### ISSUE TITLE
limit concurrent jobs on a dedicated node

### ISSUE THREAD

**Issue by fommil** [Shippable/support#1200]
The number of jobs running on a dedicated host appears to be unbounded, this can take the box down. It would be good to limit this on a per-host basis.

**Comment by manishas** [comment]
on the roadmap and coming up soon.

**Comment by rogierslag** [comment]
_Commenting to keep myself up to date_

I'd also like this feature, when the box is running 4 simultaneous builds it finished all the builds within 6 minutes. Double the number of builds and suddenly it takes 30 minutes to finish them. So I would be very happy to limit the number of builds, because in the end that causes faster PR build results

**Comment by rogierslag** [comment]
![screenshot 2015-04-22 15 55 05](https://cloud.githubusercontent.com/assets/2778689/7276136/156d8a34-e908-11e4-90ff-90742e2bfe47.png)

That is one sad dedicated host. @joostverdoorn

**Comment by avinci** [comment]
We are working on a better scheduling algo....One design construct we had was minion servers should not call home other than the queue....which ended up where state management at the minion host level became necessary which causes other issues as we need to run distributed systems across clouds. 

TLDR, we are going to have a fix in the next 2-3 weeks as we are revamping our scheduling engine.

**Comment by rogierslag** [comment]
:heart:!

**Comment by rogierslag** [comment]
I think this can be closed since dedicated hosts are not available anymore?

**Comment by manishas** [comment]
@rogierslag we are actually running a preview of a new reincarnation of dedicated hosts called single tenant CI. Anyone can sign up simply by selecting Single Tenant CI Beta from the Plan dropdown on the billing page. 

The feature in this issue is addressed and customers can select number of parallel builds. 

Let me know if you want more information. I will close this issue since this has been implemented.
